spec/migrations/20160322195653_move_switch_host_to_jointable_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_migration
+
+describe MoveSwitchHostToJointable do
+  migration_context :up do
+    let(:switch_stub) { migration_stub(:Switch) }
+    let(:host_stub) { migration_stub(:Host) }
+    let(:host_switches_stub) { migration_stub(:HostSwitch) }
+    it 'Move host-to-switch relationship to host_switches table' do
+      host = host_stub.create!
+      switch1 = switch_stub.create!(:host_id => host.id)
+      switch2 = switch_stub.create!(:host_id => host.id)
+      expect(host.switches.length).to eq 2
```

Comment:
I think this expectation is unneeded.  It's good for local testing but ultimately is testing if you did rails properly.